### PR TITLE
Added retrieve category by Id scenario

### PR DESCRIPTION
**Pull request by adrianovi**

### CODE REVIEW

File: src/test/resources/features/categories/catefories.feature [hunk 1]
```diff
@@ -27,3 +27,16 @@ Feature: Categories
     Examples:
       | User Role     | Status Line          |
       | administrator | HTTP/1.1 201 Created |
+
+  @RetrieveACategory @Smoke @Test
+  Scenario Outline: A user with proper role should be able to retrieve a post
```
Comment (eliosfg) [inline src/test/resources/features/categories/catefories.feature:32]: You are going to retrieve a 'post' or a 'category'?

**Review by eliosfg — COMMENTED** [review]
See my comment

File: src/test/resources/features/categories/catefories.feature (hunk 1 above)
Comment (Agustin-Mediotti) [inline src/test/resources/features/categories/catefories.feature]: This should return a category

Comment (adrianovi) [reply src/test/resources/features/categories/catefories.feature]: Nice!, I fixed it

**Review by Agustin-Mediotti — CHANGES_REQUESTED** [review]

File: src/test/resources/features/categories/catefories.feature (hunk 1 above)
Comment (adrianovi) [reply src/test/resources/features/categories/catefories.feature]: That's right, already fixed

**Review by Agustin-Mediotti — APPROVED** [review]
Good job

**Review by eliosfg — CHANGES_REQUESTED** [review]
I realized that your catefories.feature is wrong named.

**Pull request merged**